README.md

```diff
@@ -71,3 +71,37 @@ Update the version number in the local package.json file so that it interacts co
 
 ### Inputs
 - `version` - The new active version number of the package
+
+## [build-with-ubp](https://github.com/mindjolt/uc-actions/tree/main/build-with-ubp)
+
+Create a build using UBP, supports both UnityCore version and UC-UBP-SDK versions.
+
+### Inputs
+- `files` - The files you want to build with space delimited
+    - Files with `%PROJECT_PATH%` will have the UBP Saved Project settings path inserted
+    - Files that start with `./` will use what ever directory was last used.
+- `platform` - The UBP platform you with to build.
```

Comment:
`you with to build`? 🐃 

It would also be good to list all of the valid values for this since they are pretty specific.